### ISSUE TITLE
Support for: group_add

### ISSUE THREAD

**Issue by cdrage** [kubernetes/kompose#437]
No error / warning when using this key.

**Comment by procrypt** [comment]
@cdrage `group_add` is supported by `libcompose` but I'm having some trouble using it, opened up an issue on `libcompose`.
https://github.com/docker/libcompose/issues/446

**Comment by gitlawr** [comment]
upstream issue was fixed so I think we can take it a step further

**Comment by surajnarwade** [comment]
@cdrage @procrypt i think, `group_add` mapping doesn't make any sense in kompose, so it will be better to add it in unsupported keys, sending PR for the same :)

**Comment by cdrage** [comment]
@surajnarwade @surajssd @gitlawr 

After some investigation, you are actually able to map this to securityContext: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/

**Comment by surajnarwade** [comment]
@cdrage , for docker-compose file like,

```
version: '2'
services:
  myservice:
    image: alpine
    group_add:
      - "1004"
```

it will be,

```
apiVersion: extensions/v1beta1
kind: Deployment
metadata:
  creationTimestamp: null
  labels:
    io.kompose.service: myservice
  name: myservice
spec:
  replicas: 1
  strategy: {}
  template:
    metadata:
      creationTimestamp: null
      labels:
        io.kompose.service: myservice
    spec:
      containers:
      - image: alpine
        name: myservice
        resources: {}
      restartPolicy: Always
      securityContext:
        supplementalGroups:
        - 1004
status: {}
```

For, docker-compose we can mention group in string format but for kubernetes we need only group  id

**Comment by surajnarwade** [comment]
@cdrage is that okay ?

**Comment by cdrage** [comment]
@surajnarwade Yup. That works perfectly.

**Comment by surajnarwade** [comment]
@cdrage , awesome, sending PR for the same

**Comment by cdrage** [comment]
This has been merged in! #739
